app/src/processing/app/Base.java

```diff
@@ -1324,6 +1329,42 @@ public void selectTargetBoard(TargetBoard targetBoard) {
     onBoardOrPortChange();
     rebuildImportMenu(Editor.importMenu);
     rebuildExamplesMenu(Editor.examplesMenu);
+    try {
+      rebuildRecentBoardsMenu();
+    } catch (Exception e) {
+      // TODO Auto-generated catch block
+      e.printStackTrace();
```

Comment:
Why is this try block needed? I suppose that GUI modifications can throw, but why doesn't the same hold for e.g. `rebuildExamplesMenu()`? Does that catch exceptions internally? Should `rebuildRecentBoardsMenu()` do the same?